scripts/run_performance_tests.sh

```diff
@@ -86,7 +96,19 @@ else
   export XVFB_PREFIX="/usr/bin/xvfb-run"
 fi
 
-$XVFB_PREFIX $PYTHON_CMD scripts/backend_tests.py --test_path='core/tests/performance_tests' $@ 
+# If no arguments are given.
+if [ $# -eq 0 ]; then
+  # $XVFB_PREFIX $PYTHON_CMD scripts/backend_tests.py --test_path='core/tests/performance_tests'
+  $XVFB_PREFIX $PYTHON_CMD scripts/backend_tests.py --test_target=core.tests.performance_tests.collection_player_test
+  $XVFB_PREFIX $PYTHON_CMD scripts/backend_tests.py --test_target=core.tests.performance_tests.creator_dashboard_test
+  $XVFB_PREFIX $PYTHON_CMD scripts/backend_tests.py --test_target=core.tests.performance_tests.exploration_editor_test
+  $XVFB_PREFIX $PYTHON_CMD scripts/backend_tests.py --test_target=core.tests.performance_tests.exploration_player_test
+  $XVFB_PREFIX $PYTHON_CMD scripts/backend_tests.py --test_target=core.tests.performance_tests.library_test
+  $XVFB_PREFIX $PYTHON_CMD scripts/backend_tests.py --test_target=core.tests.performance_tests.profile_page_test
+  $XVFB_PREFIX $PYTHON_CMD scripts/backend_tests.py --test_target=core.tests.performance_tests.splash_test
+else
+  $XVFB_PREFIX $PYTHON_CMD scripts/backend_tests.py $@
```

Comment:
OK, add above this line:

# TODO(gvishal): This else branch is here so that we have a way to run individual performance tests. Modify this script to take a --test_name parameter instead, and run only that particular test in core.tests.performance_tests.